frontend/src/wailsjs/go/eventlog/EventLog.js

```diff
@@ -3,21 +3,25 @@
 // This file is automatically generated. DO NOT EDIT
 
 export function DebugEvent(arg1, arg2) {
-  return ObfuscatedCall(0, [arg1, arg2]);
```

Comment:
I didn't realize that an obfuscated build changed the local file. I'll need to watch for that.